src/cmd.rs

```diff
@@ -2343,3 +2366,97 @@ command!(
         }
     }
 );
+
+command!(
+    Jobs,
+    "jobs",
+    "Get jobs (in current namespace if set)",
+    |clap: App<'static, 'static>| clap.arg(
+        Arg::with_name("label")
+            .short("l")
+            .long("label")
+            .help("Get jobs with specified label selector")
+            .takes_value(true)
+    ).arg(
+        Arg::with_name("regex")
+            .short("r")
+            .long("regex")
+            .help("Filter jobs by the specified regex")
+            .takes_value(true)
+    ),
+    vec!["job", "jobs"],
+    noop_complete,
+    |matches, env, writer| {
+        let regex = match ::table::get_regex(&matches) {
+            Ok(r) => r,
+            Err(s) => {
+                write!(stderr(), "{}\n", s).unwrap_or(());
+                return;
+            }
+        };
+
+        let mut urlstr = if let Some(ref ns) = env.namespace {
+            format!("/apis/batch/v1/namespaces/{}/jobs", ns)
+        } else {
+            "/apis/batch/v1/jobs".to_owned()
+        };
+
+        if let Some(label_selector) = matches.value_of("label") {
+            urlstr.push_str("?labelSelector=");
+            urlstr.push_str(label_selector);
+        }
+
+        let dl: Option<JobList> = env.run_on_kluster(|k| k.get(urlstr.as_str()));
```

Comment:
I know I'm guilty of using short variable names, but I do try and have them make some sense :)  For this I'd request you use `jl` for (joblist) rather than `dl`.

Reply:
Right, I was copy pasting the deployments code and kind of missed this details.